src/test/java/seedu/address/logic/parser/EditClientCommandParserTest.java

```diff
@@ -43,40 +44,35 @@
 import seedu.address.model.product.Product;
 import seedu.address.testutil.EditClientDescriptorBuilder;
 
-public class EditCommandParserTest {
+public class EditClientCommandParserTest {
 
     private static final String PRODUCT_EMPTY = " " + PREFIX_PRODUCT;
 
-    private static final String MESSAGE_INVALID_FORMAT =
-            String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditClientCommand.MESSAGE_USAGE);
-
     private EditClientCommandParser parser = new EditClientCommandParser();
 
     @Test
     public void parse_missingParts_failure() {
         // no index specified
-        assertParseFailure(parser, VALID_NAME_AMY, MESSAGE_INVALID_FORMAT);
-
-        // no field specified
-        assertParseFailure(parser, " i/1", EditClientCommand.MESSAGE_NOT_EDITED);
+        String expectedMessage = String.format(MESSAGE_INVALID_COMMAND_FORMAT, EditClientCommand.MESSAGE_USAGE);
+        assertParseFailure(parser, VALID_NAME_AMY, expectedMessage);
 
         // no index and no field specified
```

Comment:
just a sidenote, in keeping with the heuristic to test one at a time, you might want to separate this

Reply:
okk